### ISSUE TITLE
Add gevent to web frameworks or general libraries

### ISSUE THREAD

**Issue by lewoudar** [python/python-dev-survey#9]
Hello, first of all thank you for your surveys. But since last year I did the survey, I think it is not fair that **gevent** is not even listed in "web frameworks" or "general libraries" among with tornado and Twisted. It is a library used by many "known" companies like Pinterest or Spotify to name only those there.

**Comment by filippovd20** [comment]
Agree! We should include it next time! Thanks for letting us know @lewoudar !

**Comment by lewoudar** [comment]
You are welcome 🙂

**Comment by rafil2229** [comment]
@lewoudar thanks for getting us know
